crates/corro-agent/src/agent/tests.rs

```diff
@@ -74,18 +72,15 @@ async fn insert_rows_and_gossip() -> eyre::Result<()> {
 
     let res = timeout(
         Duration::from_secs(5),
-        client.request(
-            hyper::Request::builder()
-                .method(hyper::Method::POST)
-                .uri(format!("http://{}/v1/transactions", ta1.agent.api_addr()))
-                .header(hyper::header::CONTENT_TYPE, "application/json")
-                .body(serde_json::to_vec(&req_body)?.into())?,
-        ),
+        client
+            .post(format!("http://{}/v1/transactions", ta1.agent.api_addr()))
+            .header(hyper::header::CONTENT_TYPE, "application/json")
+            .body(serde_json::to_vec(&req_body)?)
+            .send(),
```

Comment:
The difference between hyper::Client and reqwest are slight, but tend to result in smaller code.